tutorials/asr/02_Speech_Commands.ipynb

```diff
@@ -0,0 +1,1069 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "02_Speech_Commands.ipynb",
+      "provenance": [],
+      "collapsed_sections": [],
+      "toc_visible": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "accelerator": "GPU"
+  },
+  "cells": [
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "R12Yn6W1dt9t",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "\"\"\"\n",
+        "You can run either this notebook locally (if you have all the dependencies and a GPU) or on Google Colab.\n",
+        "\n",
+        "Instructions for setting up Colab are as follows:\n",
+        "1. Open a new Python 3 notebook.\n",
+        "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+        "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+        "4. Run this cell to set up dependencies.\n",
+        "\"\"\"\n",
+        "# If you're using Google Colab and not running locally, run this cell.\n",
+        "\n",
+        "## Install dependencies\n",
+        "!pip install wget\n",
+        "!apt-get install sox libsndfile1 ffmpeg\n",
+        "!pip install unidecode\n",
+        "\n",
+        "# ## Install NeMo\n",
+        "!python -m pip install --upgrade git+https://github.com/NVIDIA/NeMo.git@candidate#egg=nemo_toolkit[all]\n",
```

Comment:
Why not nemo_toolkit[asr]?

Reply:
Ahh I just used the ASR tutorial header and removed the hash, good point.

Reply:
Seemingly `tokenizers` is not part of nemo_toolkit[asr]. 

```python
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-8-327743e4f6ae> in <module>()
      3 # NeMo's ASR collection - this collections contains complete ASR models and
      4 # building blocks (modules) for ASR
----> 5 import nemo.collections.asr as nemo_asr

6 frames
/usr/local/lib/python3.6/dist-packages/nemo/collections/common/tokenizers/tokenizer_utils.py in <module>()
     16 from typing import List, Optional
     17 
---> 18 from transformers import AlbertTokenizer, BertTokenizer, DistilBertTokenizerFast, RobertaTokenizer
     19 
     20 import nemo

ModuleNotFoundError: No module named 'transformers'

---------------------------------------------------------------------------
NOTE: If your import is failing due to a missing package, you can
manually install dependencies using either !pip or !apt.

To view examples of installing some common dependencies, click the
"Open Examples" button below.
---------------------------------------------------------------------------
```